index.js

```diff
@@ -4,19 +4,38 @@
 module.exports = {
   name: 'test-selectors',
 
-  setupPreprocessorRegistry: function(type, registry) {
-    var appOptions = registry.app.options || {};
+  _assignOptions: function(app) {
+    var ui = app.project.ui;
+
+    var appOptions = app.options || {};
     var addonOptions = appOptions['ember-test-selectors'] || {};
-    var environments = addonOptions.environments || ['production'];
 
-    if (environments.indexOf(registry.app.env) !== -1) {
-      var StripTestSelectorsTransform = require('./strip-test-selectors');
+    if (addonOptions.environments) {
+      ui.writeDeprecateLine('The "environments" option in "ember-test-selectors" has been replaced ' +
+        'with a "strip" option. Use e.g. "strip: EmberApp.env() === \'production\'" instead to ' +
```

Comment:
*the* would be more correct in my opinion, because it matches the *the* from the beginning of the sentence.